src/app/components/event-item/controls/comment-add-placeholder/placeholder/placeholder.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+	<div class="form-control -container" @click="onClick('')">
+		<div class="-placeholder">
+			<em>Leave a comment...</em>
+		</div>
+		<div class="-buttons" @click.stop="onClick('gif')">
```

Comment:
Why do we show the gif in the preview here anyways? we usually on show these controls when the editor is focused anyways.